docs/shared/packages/angular/angular-nx-version-matrix.md

```diff
@@ -1,6 +1,6 @@
 # Nx and Angular Versions
 
-The latest version of Nx supports the last two major versions of Angular (current and previous). Workspaces in any of those versions are recommended to use the latest version of Nx to benefit from all the new features and fixes.
+The latest version of Nx supports the [LTS versions of Angular](https://angular.io/guide/releases#actively-supported-versions). Workspaces in any of those versions are recommended to use the latest version of Nx to benefit from all the new features and fixes.
```

Comment:
Doesn't this imply that Nx does NOT support the current version "a.k.a Active" (16 as of now). 16 is not going to be LTS until 17 is released. Is this the intention, to only support it while in LTS stage?

Reply:
Good catch, thanks!